cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -33,6 +33,10 @@ data:
   # Supported values from 0 to 5. 0 for general useful logs, 5 for trace level verbosity.
   # CSI_LOG_LEVEL: "0"
 
+  # By default OMAP generator sidecar is deployed with CSI provisioner pod, to disable
+  # it set it to false.
```

Comment:
Why would someone want to disable the omap generator? Is there a doc link to csi that will explain this feature so users can know whether they want it running?

Reply:
Seconded. More docs about why OMAP should be enabled/disabled are good.

Reply:
Added a comment. we are working on a new controller for automating Failover and failback scenarios. Currently, it's an experimental feature and will be complete in the next releases.  We are working on the documentation part will be done in separate PR's.

Reply:
Ok, this comment helps for now, and since it's disabled by default it's less urgent for the docs.